campbot/checkers.py

```diff
@@ -486,3 +487,24 @@ def __init__(self):
 
     def test_contribution(self, old_doc, new_doc):
         return old_doc.quality == new_doc.quality
+
+class TitleLevel(object):
+    def __init__(self):
+        self.name = "Titre de niveau trop élevé"
+        self.fail_marker = emoji("/uploads/default/original/2X/0/0178043b1b70e669946f609571bd4b8f7d18e820.png",
```

Comment:
Use another icon. you can use HTML. Maybe `<strong>H</strong>` for fail_marker, and `<ins><strong>H</strong></ins>` for success_marker ?